docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -0,0 +1,266 @@
+==============================================================
+Upgrade Guide - Scylla Monitoring 3.x to Scylla Monitoring 4.y
+==============================================================
+
+This document is a step by step procedure for upgrading `Scylla Monitoring Stack </operating-scylla/monitoring/3.0>`_ from version 3.x to 3.y, for example, between 3.0 to 3.0.1.
```

Comment:
Is that correct? Later on, we tell the user to install 4.y.
I THINK it should be:
This document is a step-by-step procedure for upgrading `Scylla Monitoring Stack </operating-scylla/monitoring/3.0>`_ from version 3.x to 4.y.